package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "framer-motion": "^7.3.5",
+    "lucide-react": "^0.453.0",
```

Comment:
Don't add another library. 

You can use react-icons itself for the Sun and Moon.